lib/sparql_to_sw_solr/instance_solr_doc.rb

```diff
@@ -25,14 +33,62 @@ def solr_doc_hash
         return if CKEY_BLACKLIST.include?(@ckey)
         doc = {}
         doc[:id] = instance_uri_to_ckey
-        doc
+        build_doc_titles(doc)
       end
     end
 
     private
 
+    def build_doc_titles(doc)
+      doc[:id] = instance_uri_to_ckey
+      doc[:title_245a_search] = primary_triples(['mainTitle'], title_query_str)
+      doc[:title_245_search] = primary_triples(['rdf-schema#label'], title_query_str)
```

Comment:
I would rather see this as a concatenation of "mainTitle" and "subtitle" than rely on the label -- I would like to relay on labels as little as possible ... is that practical?